src/main/java/core/basesyntax/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.report;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {

```

Comment:
The class is missing a package declaration. According to the checklist, the logic should be organized in packages. Consider placing this class in a package that groups it with other similar classes.